### ISSUE TITLE
Generated docs for TF 2.1 and TF 2.2 are missing information present in the source docs

### ISSUE THREAD

**Issue by foxik** [tensorflow/tensorflow#37610]
## URL(s) with the issue:

The issue affects many pages, here is one example:
- TF 2.2: https://www.tensorflow.org/versions/r2.2/api_docs/python/tf/keras/optimizers/schedules/ExponentialDecay
- TF 2.1: https://www.tensorflow.org/versions/r2.1/api_docs/python/tf/keras/optimizers/schedules/ExponentialDecay
- TF 2.0: https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/optimizers/schedules/ExponentialDecay

## Description of issue (what needs changing):
The generated docs for TF 2.1 and TF 2.2 is missing important information. Notably, the whole documentation of `__init__` method containing information like detailed learning rate computation
```python
def decayed_learning_rate(step):
  return initial_learning_rate * decay_rate ^ (step / decay_steps)
```
is missing in TF 2.1 and TF 2.2. However, the information is still present in the source, see 
https://github.com/tensorflow/tensorflow/blob/3c1e8c03419266bb6ba379d303d3e03a380617a8/tensorflow/python/keras/optimizer_v2/learning_rate_schedule.py#L64-L134

## Further examples
For example Adam optimizer is also affected, see
- TF 2.2: https://www.tensorflow.org/versions/r2.2/api_docs/python/tf/keras/optimizers/Adam
- TF 2.0: https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/keras/optimizers/Adam
The TF 2.0 version contains a lot of math describing how Adam works, which is not present in TF 2.2 docs.

**Comment by yashk2810** [comment]
Thanks for reporting. It will be fixed in the nightly version of the API docs tomorrow.

**Comment by foxik** [comment]
It is indeed fixed in the nightly, thanks!
